types/artifacts.d.ts

```diff
@@ -82,6 +82,12 @@ declare global {
       ScriptElements: Array<Artifacts.ScriptElement>;
       /** The dimensions and devicePixelRatio of the loaded viewport. */
       ViewportDimensions: Artifacts.ViewportDimensions;
+      /** All the form elements in the page and formless inputs. */
+      Forms: Artifacts.Form[];
+      /** Attributes of an input element. */
+      FormInputs: Artifacts.FormInputsType;
```

Comment:
these aren't gatherers AFAICT, is there something in particular you were trying to solve by adding these here?